app/src/main/kotlin/com/simplemobiletools/keyboard/media/emoji/Emoji.kt

```diff
@@ -0,0 +1,8 @@
+package com.simplemobiletools.keyboard.media.emoji
+
+
+data class Emoji(val value: String, val name: String, val keywords: List<String>) {
+    override fun toString(): String {
+        return "Emoji { value=$value, name=$name, keywords=$keywords }"
```

Comment:
I dont think we need this as it is a data class

Reply:
check pls